### DISCUSSION TITLE
1panel的WAF拦截了Alist的保存设置的请求

### DISCUSSION THREAD

**Discussion by xiaozhu52055** [1Panel-dev/1Panel#4846] [Q&A]
写好全局里的自定义内容后点保存出现
Request failed with status code 403
log：
"POST /api/admin/setting/save HTTP/2.0" 403 16910 "https://pan.XXXX.cn/@manage/settings/global"
测试关闭WAF后正常，请问怎么修改WAF的规则，使其正常？

**Comment by zhengkunwang223** [comment]
给自己的 IP 加白名单 
或者给这个 URL  加白名单
